projects/storefrontlib/layout/direction/direction.service.ts

```diff
@@ -81,7 +81,7 @@ export class DirectionService implements OnDestroy {
    * Sets the direction attribute for the given element. If the direction is undefined, the `dir`
    * attribute is removed.
    */
-  setDirection(el: HTMLElement, direction: DirectionMode): void {
+  setDirection(el: HTMLElement, direction?: DirectionMode): void {
```

Comment:
I don't think we should make direction optional. Similar reason as [this](https://github.com/SAP/spartacus/pull/15658/files#r870749898)